config.fish

```diff
@@ -13,9 +13,19 @@ end
 set -U __done_min_cmd_duration 10000
 set -U __done_notification_urgency_level low
 
-## Enable Wayland support for Mozilla app if session type is Wayland
+## Enable Wayland support for different applications
 if [ "$XDG_SESSION_TYPE" = "wayland" ]
+    set -gx WAYLAND 1
+    set -gx QT_QPA_PLATFORM 'wayland;xcb'
+    set -gx GDK_BACKEND wayland
```

Comment:
That should be the default, no?

Reply:
Better to set anyway the flag for only wayland, sometimes it doesn't put that flags at automatically.